app/code/Magento/Eav/Service/PopulateOptionsWithAdditionalData.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Eav\Service;
```

Comment:
Hello @engcom-Golf, 
I may be wrong but Magento usually uses the `Model` namespace for service classes so I would suggest adhering to this convention.
What do you think?

Reply:
Hi @aleron75,
agree with you, updated solution